t/math/compact_dual.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?latexml class="article"?>
+<?latexml package="latexml"?>
+<?latexml RelaxNGSchema="LaTeXML"?>
+<document xmlns="http://dlmf.nist.gov/LaTeXML">
+  <resource src="LaTeXML.css" type="text/css"/>
+  <resource src="ltx-article.css" type="text/css"/>
+  <para xml:id="p1">
+    <p><Math mode="inline" tex="\@CSYMBOL{power}x2+\@CSYMBOL{power}y3" text="power@(x, 2) + power@(y, 3)" xml:id="p1.m1">
```

Comment:
Btw, an unexpected aside, it looks like `\@APPLY` and `\DUAL` don't have a reversion back into the `tex` attribute.

Reply:
Learning more as I go along, the `tex` attribute is dedicated to the presentation TeX, which I've know obeyed by passing the `hide_content_reversion` option to `\DUAL`. Added a commit.

Feels a little crutchy, should be automatable in the long run... Similarly the `content_tex` attribute needs what a TeX.pool comment referred to as:
```
# NOTE: work through this systematically!
```

and is currently not ideal in my latest XML file. It also reminded me of the old discussion around tex having two annotations for the source TeX ( #432 ), just linking it in case someone's picking up breadcrumbs later on. Not something I want to jump into right now, but I ended up noticing... The improved XMDual pruning is the only focus here, sorry for the diversion.